applications/ContactStructuralMechanicsApplication/custom_utilities/process_factory_utility.h

```diff
@@ -56,9 +56,9 @@ class ProcessFactoryUtility
     /// Counted pointer of ProcessFactoryUtility
     KRATOS_CLASS_POINTER_DEFINITION( ProcessFactoryUtility );
 
-    typedef typename boost::python::object ObjectType;
+    typedef boost::python::object ObjectType;
```

Comment:
can i ask you why you took out the typename? did u try if it compiles in windows without?

Reply:
Indeed this is corrrected in y branch, I will correct here too

Reply:
In fact is corrected here, the typename is red->removed